server/src/main/resources/c4_component.puml

```diff
@@ -1,3 +1,10 @@
+' convert it with additional command line argument -DRELATIVE_INCLUDE="." to use locally
+!if %variable_exists("RELATIVE_INCLUDE")
+  !include %get_variable_value("RELATIVE_INCLUDE")/C4_Container.puml
+!else
+  !include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+!endif
+
```

Comment:
We should not include this part, could you please remove it?

Reply:
Removed.